database/abusescanner_test.go

```diff
@@ -0,0 +1,150 @@
+package database
+
+import (
+	"context"
+	"io/ioutil"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+// TODO: needs to be extended to cover all methods
+
+const (
+	// mongoTestUsername is the username for the test database
+	mongoTestUsername = "admin"
+
+	// mongoTestPassword is the password for the test database
+	mongoTestPassword = "aO4tV5tC1oU3oQ7u"
+
+	// mongoTestConnString is the connection string for the test database
+	mongoTestConnString = "mongodb://localhost:37017"
```

Comment:
PP: I liked that these were exposed in testing because we could have reused them in, say, `parser_test.go` instead of duplicating them. But this is a minor thing, I just wanted to note my preference.